### ISSUE TITLE
Non-existent libraries are added to the user library list

### ISSUE THREAD

**Issue by allthom** [codefori/vscode-ibmi#1879]
👉🏻Hello everyone, while I was manually adding libraries to my user library list I misspelled the name of one of these libraries. I wrote TETE01 instead of TEST01, there is no TETE01 on my system, nevertheless TETE01 has been added to the list on the right.

<hr />

|Context|Version|
|-|-|
|Code for IBM i version|2.7.0|
|Visual Studio Code version|1.86.2|
|Operating System|win32_x64|

<details>
<summary>Active extensions</summary>

```
COBOL (cobol): 10.2.21
Code for IBM i Walkthroughs (vscode-ibmi-walkthroughs): 0.4.0
Db2 for IBM i (vscode-db2i): 0.7.1
ESLint (vscode-eslint): 2.4.4
Emmet (emmet): 1.0.0
Error Lens (errorlens): 3.16.0
Git (git): 1.0.0
Git Base (git-base): 1.0.0
GitHub (github): 0.0.1
IBM i Notebooks (vscode-ibmi-notebooks): 0.0.6
JSON Language Features (json-language-features): 1.0.0
Merge Conflict (merge-conflict): 1.0.0
Node Debug Auto-attach (debug-auto-launch): 1.0.0
TODO Highlight (vscode-todo-highlight): 1.0.5
TypeScript and JavaScript Language Features (typescript-language-features): 1.0.0
WSL (remote-wsl): 0.86.0
```
</details>

<hr />

<details>
<summary>Remote system</summary>

|Setting|Value|
|-|-|
|IBM i OS|V7R3M0|
|Tech Refresh|12|
|CCSID|65535|
|Default CCSID|280|
|SQL|Disabled
|Source dates|Disabled

### Enabled features

|/usr/bin|/QSYS.lib/ILEDITOR.lib|/QSYS.LIB|
|-|-|-|
|attr|GENCMDXML.PGM|QZDFMDB2.PGM|
|iconv|GETNEWLIBL.PGM||
|ls|||
|setccsid|||
|tar|||
</details>

<details>
<summary>Shell env</summary>

```bash
```
</details>

<details>
<summary>Variants</summary>

```json
{
  "american": "#@$",
  "local": "��$"
}
```
</details>

<details>
<summary>Errors</summary>

```json
[
  {
    "command": "/QOpenSys/usr/bin/qsh",
    "code": 1,
    "stderr": "liblist: 001-0081 Errore CPF2110 rilevato aggiungendo la libreria WRONGLIB all'elenco librerie.",
    "cwd": "/home/MYUSER"
  },
  {
    "command": "/QOpenSys/usr/bin/qsh",
    "code": 1,
    "stderr": "liblist: 001-0081 Errore CPF2110 rilevato aggiungendo la libreria WRONGLIB all'elenco librerie.",
    "cwd": "/home/MYUSER"
  },
  {
    "command": "export BUILDLIB=\"QGPL\" && export CURLIB=\"QGPL\" && export USERNAME=\"MYUSER\" && export HOST=\"172.22.46.4\" && export HOME=\"/home/MYUSER\" && export WORKDIR=\"/home/MYUSER\" && export LIBLS=\"a_very_long_list\" && env",
    "code": 1,
    "stderr": "bsh: BUILDLIB=QGPL: is not an identifier",
    "cwd": "/home/MYUSER"
  }
]
```
</details>

**Comment by worksofliam** [comment]
Please share the contents of your Code for IBM i output channel after you add the invalid library to the list.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/3708366/178739532-03c8ad30-84de-4698-9904-65938f2ef651.png">

**Comment by allthom** [comment]
.: echo "This should be the only text!"
{
    "code": 0,
    "signal": null,
    "stdout": "This should be the only text!",
    "stderr": ""
}

.: echo $HOME && cd && test -w $HOME
{
    "code": 0,
    "signal": null,
    "stdout": "/home/MYUSER",
    "stderr": ""
}

/home/MYUSER: /QOpenSys/usr/bin/qsh
liblist
{
    "code": 0,
    "signal": null,
    "stdout": "MAXSYS      SYS\nQSYS        SYS\nQSYS2       SYS\nQHLPSYS     SYS\nQUSRSYS     SYS\nQSHELL      PRD\nQGPL        USR\nQTEMP       USR\nJGPL        USR\nACGGAA      USR",
    "stderr": ""
}

/home/MYUSER: /QOpenSys/usr/bin/qsh
system  "CRTLIB LIB(ILEDITOR) TEXT('Code for i temporary objects. May be cleared.')"
{
    "code": 1,
    "signal": null,
    "stdout": "",
    "stderr": "CPC9801:  Oggetto MXCMD2RUN di tipo *DTAQ creato nella libreria QTEMP.\nCPC9801:  Oggetto MXRSNDMG di tipo *DTAQ creato nella libreria QTEMP.\nCPF2111:  La libreria ILEDITOR esiste gi�."
}

/home/MYUSER: [ -d "/tmp" ]
{
    "code": 0,
    "signal": null,
    "stdout": "",
    "stderr": ""
}


User CCSID is 65535; falling back to using default CCSID 280

/home/MYUSER: LC_ALL=EN_US.UTF-8 system "call QSYS/QZDFMDB2 PARM('-d' '-i' '-t')"
Call QSYS2.QCMDEXC('CHGJOB CCSID(280)');
select OS_VERSION concat '.' concat OS_RELEASE as VERSION from sysibmadm.env_sys_info
/home/MYUSER: LC_ALL=EN_US.UTF-8 system "call QSYS/QZDFMDB2 PARM('-d' '-i' '-t')"
Call QSYS2.QCMDEXC('CHGJOB CCSID(280)');
select max(ptf_group_level) as HIGHEST_DB2_PTF_GROUP_LEVEL from qsys2.group_ptf_info where PTF_GROUP_DESCRIPTION like 'DB2 FOR IBM I%' and ptf_group_status = 'INSTALLED';
/home/MYUSER: ls $HOME/.vscode/codeforiserver-1.4.9.jar
{
    "code": 0,
    "signal": null,
    "stdout": "/home/MYUSER/.vscode/codeforiserver-1.4.9.jar",
    "stderr": ""
}

{
    "code": 0,
    "signal": null,
    "stdout": "DB2>\nDB20000I  THE SQL COMMAND COMPLETED SUCCESSFULLY.\nDB2>\n  ?>\n\nVERSION                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          \n---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n7.3                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              \n\n  1 RECORD(S) SELECTED.",
    "stderr": ""
}

{
    "code": 0,
    "signal": null,
    "stdout": "DB2>\nDB20000I  THE SQL COMMAND COMPLETED SUCCESSFULLY.\nDB2>\n\nHIGHEST_DB2_PTF_GROUP_LEVEL \n----------------------------\n                          29\n\n  1 RECORD(S) SELECTED.\n\nDB2>",
    "stderr": ""
}

/home/MYUSER: /QOpenSys/usr/bin/qsh
liblist -d QGPL QTEMP JGPL ACGGAA; liblist -a NONEXLIB
{
    "code": 1,
    "signal": null,
    "stdout": "",
    "stderr": "liblist: 001-0081 Errore CPF2110 rilevato aggiungendo la libreria NONEXLIB all'elenco librerie."
}

/home/MYUSER: /QOpenSys/usr/bin/qsh
liblist -d QGPL QTEMP JGPL ACGGAA; liblist -a QTEMP; liblist -a ACG_LNDDV; liblist -a ACG_LNDFC; liblist -a ACG_LNDIS; liblist -a ICOMUNE; liblist -a NUOVA; liblist -a JFGCGRPDAT; liblist -a JFGCLNDDAT; liblist -a LNDDIGF; liblist -a JFGC_PTF38; liblist -a JFGC_PTF; liblist -a JFGC_OBJ; liblist -a LNDDIG; liblist -a LND_DAT; liblist -a LND_OBJ; liblist -a ACG_OBJV3; liblist -a ACGGAA; liblist -a QGPL; liblist -a JGPL; liblist -a ACG_CUC; liblist -a WRNOGLIB; liblist -a WRONGLIB2; liblist -a NONEXLIB
{
    "code": 1,
    "signal": null,
    "stdout": "",
    "stderr": "liblist: 001-0081 Errore CPF2110 rilevato aggiungendo la libreria WRNOGLIB all'elenco librerie.\nliblist: 001-0081 Errore CPF2110 rilevato aggiungendo la libreria WRONGLIB2 all'elenco librerie.\nliblist: 001-0081 Errore CPF2110 rilevato aggiungendo la libreria NONEXLIB all'elenco librerie."
}

![immagine](https://github.com/codefori/vscode-ibmi/assets/141107907/1252d6af-ba29-4c11-840e-6b075ca95196)

**Comment by worksofliam** [comment]
@allthom A little update for you: we know the bug and we will be working on a fix for 2.9.0.
